homeassistant/components/husqvarna_automower/button.py

```diff
@@ -25,15 +25,16 @@ async def async_setup_entry(
     """Set up button platform."""
     coordinator = entry.runtime_data
     async_add_entities(
-        AutomowerButtonEntity(mower_id, coordinator) for mower_id in coordinator.data
+        AutomowerButtonEntity(mower_id, coordinator)
```

Comment:
Is this really an entity that is of such usage for users that it's worth enabling it by default?

Reply:
Yes, I think it's quite useful, because the mower got stuck sometimes (or more often if your lawn is difficult and complex), and you can create an automation to get the mower working again.

Reply:
Ok. Just for awareness; for those who already have the integration this change doesn't enable the button. It's only for new installations.